ignition/chain_config.json

```diff
@@ -34,241 +34,273 @@
       },
       "fee_params": {
         "V1": {
-          "gas_price_factor": 92,
-          "gas_per_byte": 63
+          "gas_price_factor": 92000,
+          "gas_per_byte": 233
         }
       },
-      "chain_id": 0,
+      "chain_id": 9889,
       "gas_costs": {
-        "V1": {
+        "V4": {
           "add": 2,
           "addi": 2,
-          "aloc": 2,
           "and": 2,
           "andi": 2,
-          "bal": 86,
+          "bal": 274,
           "bhei": 2,
           "bhsh": 2,
-          "burn": 25770,
+          "burn": 7566,
           "cb": 2,
-          "cfei": 2,
           "cfsi": 2,
           "div": 2,
           "divi": 2,
-          "eck1": 3114,
-          "ecr1": 42270,
-          "ed19": 2878,
+          "eck1": 1489,
+          "ecr1": 20513,
           "eq": 2,
           "exp": 2,
           "expi": 2,
-          "flag": 1,
+          "flag": 2,
           "gm": 2,
           "gt": 2,
-          "gtf": 12,
+          "gtf": 3,
           "ji": 2,
           "jmp": 2,
           "jne": 2,
           "jnei": 2,
           "jnzi": 2,
-          "jmpf": 1,
-          "jmpb": 1,
-          "jnzf": 1,
-          "jnzb": 1,
-          "jnef": 1,
-          "jneb": 1,
+          "jmpf": 2,
+          "jmpb": 2,
+          "jnzf": 2,
+          "jnzb": 2,
+          "jnef": 2,
+          "jneb": 2,
           "lb": 2,
-          "log": 165,
+          "log": 80,
           "lt": 2,
           "lw": 2,
-          "mint": 29024,
+          "mint": 6566,
           "mlog": 2,
           "mod": 2,
           "modi": 2,
           "move": 2,
-          "movi": 2,
-          "mroo": 4,
+          "movi": 1,
+          "mroo": 3,
           "mul": 2,
           "muli": 2,
           "mldv": 3,
           "noop": 1,
           "not": 2,
-          "or": 2,
+          "or": 1,
           "ori": 2,
-          "poph": 3,
-          "popl": 3,
-          "pshh": 4,
-          "pshl": 4,
-          "ret_contract": 134,
-          "rvrt_contract": 153,
+          "poph": 2,
+          "popl": 2,
+          "pshh": 3073,
```

Comment:
why did these get so much more expensive?

Reply:
Before benchmarks had a bug, which caused small values for this opcode. Now we also charge for stack growth